.github/workflows/autogenerated/test-find-that-file.yml

```
@@ -0,0 +1,25 @@
+name: test problem (find-that-file)
+
+on:
+  pull_request:
+    paths:
+      - "packages/problems/find-that-file/**"
+
+jobs:
+  test:
```

actionlint (1.7.4)

job "test" needs job "test-entrypoint" which does not exist in this workflow

(job-needs)